src/components/VideoModal/index.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import ButtonStyles from "../../styles/Buttons.module.scss";
+import Close from "../Images/Icons/Close";
+import Styles from "./styles.module.scss";
+
+export default function VideoModal({ closeClick, closeModal }) {
+  window.onkeydown = function onEscKey(e) {
```

Comment:
I think we need to wrap this in `useEffect` to make sure we don't keep adding the event listener or accidentally remove it somewhere else. I can help with this if you like, `useEffect` is really damn confusing!